pkg/metrics/cluster/node_test.go

```diff
@@ -169,7 +171,7 @@ func startNode(t *testing.T, srv agentproto.ScrapingServiceServer) {
 	lcConfig.Addr = l.Addr().(*net.TCPAddr).IP.String()
 	lcConfig.Port = l.Addr().(*net.TCPAddr).Port
 
-	lc, err := ring.NewLifecycler(lcConfig, ring.NewNoopFlushTransferer(), "agent", "agent", false, prometheus.NewRegistry())
+	lc, err := ring.NewLifecycler(lcConfig, ring.NewNoopFlushTransferer(), "agent", "agent", false, logger, prometheus.WrapRegistererWithPrefix("cortex_", prometheus.NewRegistry()))
```

Comment:
Actually in thinking, you can pass `nil` here in place of the registry since we don't even save it off to a variable, so I don't think we even look at it.